src/commands/aqua.ts

```diff
@@ -245,7 +246,7 @@ const resolveAbsoluteAquaPath = async ({
     return resolve(projectRootDir, maybePathFromFluenceYaml);
   }
 
-  const pathFromUserInput = await input(inputArg);
+  const pathFromUserInput = await input({ ...inputArg, default: cwd() });
```

Comment:
Can you please add a comment here on what this means?

I would expect default to be `projectRootDir` so that commands do not depend on the cwd, but apparently intention here is different.

Reply:
by default input for aqua is set in fluence.yaml. If it's not set (you removed it for some reason from fluence.yaml) or you don't have fluence.yaml at all - then there is need to provide path to the input for aqua compilation. You can provide path using a flag or you will be prompted and making cwd default in this case seems reasonable. But you can still input path that you need by hand. This behaviour didn't change for a while, I am just changing the way default's are displayed/handled in all prompts.

Reply:
can you explain that in a comment to this line please?

Reply:
sure